events/bingo/bingoCardApp/bingoCard_app.js

```diff
@@ -118,7 +118,7 @@
     /* FIND OR CREATE BINGO CARD:  Search the google sheet for the user's name to see if they have already been assigned 
     numbers for this round. If they have numbers already, retrieve them. If they do not have numbers, get new ones
     and add one to the player counter text. */
-    var SPREADSHEET_URL = Script.require(Script.resolvePath('../secrets/bingoSheetURL.json?0')).sheetURL;
+    var REQUEST_URL = Script.require(Script.resolvePath('../secrets/secrets.json?0')).requestURL;
```

Comment:
I had a question about this one, I'm not for sure on this one, but I thought that Script.require takes care of resolving the path.  Do you know for sure if that isn't true?

Reply:
i don't know.